plugin/trino-exchange/src/main/java/io/trino/plugin/exchange/FileSystemExchangeSink.java

```diff
@@ -126,11 +132,20 @@ private void flushIfNeeded(ExchangeStorageWriter writer, SliceOutput buffer, boo
         }
     }
 
+    private void swapBuffer(int partitionId)
+    {
+        writeBuffers.putIfAbsent(partitionId, Slices.allocate(exchangeStorage.getWriteBufferSizeInBytes()).getOutput());
```

Comment:
computeIfAbsent

Reply:
Question: what difference does it make here?